rofl-containers/src/main.rs

```diff
@@ -0,0 +1,44 @@
+use std::env;
+
+use base64::prelude::*;
+use oasis_runtime_sdk::{cbor, modules::rofl::app::prelude::*};
+
+/// UNIX socket address where the REST API server will listen on.
+const ROFL_APPD_ADDRESS: &str = "unix:/run/rofl-appd.sock";
+
+/// A generic container-based ROFL application.
```

Comment:
Am I correct that there's nothing really container specific/related in this app at the moment. It just an app that starts the rofl-appd server (which exposes the kms via a rest api)? 

Are there any docs for future plans / goals of this app?

Reply:
Will note this in the docs.

Reply:
Added a section to the file documentation comment.

Reply:
Looks good, thanks! 👍